modules/openxr/editor/openxr_action_map_editor.cpp

```diff
@@ -387,6 +415,22 @@ void OpenXRActionMapEditor::_clear_action_map() {
 	}
 }
 
+void OpenXRActionMapEditor::register_interaction_profile_editor(const String &p_for_path, const String &p_editor_class) {
+	interaction_profile_editors[p_for_path] = p_editor_class;
+}
+
+void OpenXRActionMapEditor::register_binding_modifier_editor(const String &p_binding_modifier_class, const String &p_editor_class) {
+	binding_modifier_editors[p_binding_modifier_class] = p_editor_class;
+}
+
+String OpenXRActionMapEditor::get_binding_modifier_editor_class(const String &p_binding_modifier_class) {
+	if (binding_modifier_editors.has(p_binding_modifier_class)) {
+		return binding_modifier_editors[p_binding_modifier_class];
+	}
+
+	return "OpenXRBindingModifierEditor";
```

Comment:
How about `OpenXRBindingModifierEditor::get_class_name()` to avoid the use of magic strings and provide compilation safety.